### ISSUE TITLE
[GraphQL] ExistsFilter does not work

### ISSUE THREAD

**Issue by IonBazan** [api-platform/core#1857]
`ExistsFilter` does not work with GraphQL. 

Minimal example:
```
{
  user(posts:{exists: true}) {
    edges {
      node {
        id
        username
      }
    }
  }
}
```
Returns all users, not only those, who have any posts.

This is because the filter expects string `"true"|"false"` or integer `1|0`. However, GraphQL expects a boolean there.
The fix seems easy: adding `true` and `false` to https://github.com/api-platform/core/blob/3d17f85c79b8620901e1505a3564021a0382e4ce/src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php#L78-L80
I will try to provide a PR when I add some test to these cases.

**Comment by Simperfit** [comment]
@IonBazan Does it trigger a notice for you ?

**Comment by IonBazan** [comment]
@Simperfit GraphQL does not accept a string there. Passing `exists: "true"` triggers an error while passing `exists: true` is being ignored by lines above.

Replacing:
```php
\in_array($value[self::QUERY_PARAMETER_KEY], ['true', '1', '', null], true)
```
with
```php
\in_array($value[self::QUERY_PARAMETER_KEY], [true, 'true', '1', '', null], true)
```

Fixes the problem.

**Comment by IonBazan** [comment]
Fixed by #1861
